src/components/expanded-state/chart/chart-data-labels/ChartPercentChangeLabel.js

```diff
@@ -46,11 +43,8 @@ export default function ChartPercentChangeLabel({
   const { originalY, data } = useChartData();
   const { colors } = useTheme();
 
-  const firstValue = useSharedValue(data?.points?.[0]?.y);
-  const lastValue = useSharedValue(data?.points?.[data.points.length - 1]?.y);
-
   const defaultValue =
-    data?.points.length === 0
+    !data?.points || data?.points.length === 0
```

Comment:
so if `data` is falsy, then this is truth? Can this happen?


```suggestion
    data?.points?.length === 0 ?? true
```

Reply:
I guess it's better to use just `!data?.points.length`